g2p/apply.py

```diff
@@ -83,12 +129,22 @@ def run(self):
                         "-m",
                         self.g2p_model.get_path(),
                         "-a",
-                        self.word_list.get_path(),
+                        word_list_path,
                     ],
                     stdout=out,
                     stderr=err,
                 )
 
+    def merge(self):
+        g2p_lex_file_list = " ".join(map(tk.uncached_path, self.out_g2p_lexicon_parts))
```

Comment:
Please do not use `tk.uncached_path`, we want to get rid of this. Just use `variable.get_path()`.

But related to a statement to above, maybe you do not even need Paths here, but just regular strings to refer to intermediate files in the work folder.

Reply:
Yes, I changed it to do it in this way now.